pkg/node/kube-proxy.go

```diff
@@ -106,9 +108,19 @@ func (s *ProxyOptions) Run(ctx context.Context, ready chan<- struct{}, stopped c
 		klog.Infof("%s is ready", s.Name())
 		close(ready)
 	}()
-	if err := s.options.Run(); err != nil {
-		klog.Fatalf("%s failed to start", s.Name(), err)
```

Comment:
returning an error to stop other stuff depends on:

https://github.com/redhat-et/microshift/pull/575

We shouldn't exit so harshly without letting etcd at least shut down properly

Reply:
oh, you're right @copejon thanks for the review. I'll fix this

Reply:
ehhm, my changes didnd't go in... let me fix this, sorry

Reply:
ok, now. yes...